FFXIVClientStructs/FFXIV/Client/LayoutEngine/LayoutWorld.cs

```diff
@@ -36,4 +36,7 @@ public unsafe partial struct LayoutWorld {
 
     [MemberFunction("48 89 5C 24 ?? 48 89 74 24 ?? 57 48 81 EC ?? ?? ?? ?? 48 8B 05 ?? ?? ?? ?? 48 33 C4 48 89 84 24 ?? ?? ?? ?? 48 83 B9 ?? ?? ?? ?? ?? 48 8B F1"), GenerateStringOverloads]
     public partial byte* ResolveRsvString(byte* rsvString);
+
+    [MemberFunction("40 53 55 56 57 41 54 41 55 41 56 41 57 48 81 EC ?? ?? ?? ?? 48 8B 05 ?? ?? ?? ?? 48 33 C4 48 89 84 24 ?? ?? ?? ?? 4D 8B E8")]
+    public partial bool AddRsvString(byte* rsvString, byte* resolvedString, nint resolvedStringSize);
```

Comment:
`nint` should be `nuint` 
![c/c++ def](https://screenshots.wildwolf.dev/msedge_48b5cabf08.png)
https://open-std.org/JTC1/SC22/WG14/www/docs/n1256.pdf Chapter 7.17 Paragraph 2

Reply:
```suggestion
    public partial bool AddRsvString(byte* rsvString, byte* resolvedString, nuint resolvedStringSize);
```